{"message":"token error","time":1695048810607,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695051016634,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412668060,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412674736,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412674742,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412696706,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412696710,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412729599,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412738072,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412796648,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413071754,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413119871,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413554355,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413620570,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"token error","time":1695416419153,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416419172,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416424443,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416424453,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"Wrong password","time":1695416437149,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"Wrong password","time":1695416442127,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"Wrong password","time":1695416445441,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"token error","time":1695416446746,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416446751,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"this email were already exist","time":1695416472213,"code":409,"stackTrace":{"error":"NotUniqueEmail: this email were already exist\n    at _callee2$ (/server/src/controllers/userController.js:52:12)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/userController.js:9:103)\n    at _throw (/server/src/controllers/userController.js:11:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695734349701,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695734371006,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695734371013,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"Wrong password","time":1695734378152,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"need token","time":1695735120460,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"user with this data didn`t exist","time":1695735127323,"code":404,"stackTrace":{"error":"UserNotFoundError: user with this data didn`t exist\n    at _callee2$ (/server/src/controllers/queries/userQueries.js:19:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"need token","time":1695735128330,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695735128336,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695737475238,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737475280,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737475291,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477830,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477835,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477841,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480894,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480899,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480904,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480910,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"need token","time":1695737670412,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695737672270,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695737672275,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741874040,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741874056,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741879504,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741879508,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
